README.md

```diff
@@ -89,19 +89,19 @@ and Grafana using the following `Furyfile.yml` :
 ```yaml
 bases:
   - name: monitoring/prometheus-operator
-    version: v1.5.0
+    version: v1.6.0-rc1
   - name: monitoring/prometheus-operated
-    version: v1.5.0
+    version: v1.6.0-rc1
   - name: monitoring/alertmanager-operated
-    version: v1.5.0
+    version: v1.6.0-rc1
   - name: monitoring/node-exporter
-    version: v1.5.0
+    version: v1.6.0-rc1
   - name: monitoring/kube-state-metrics
-    version: v1.5.0
+    version: v1.6.0-rc1
   - name: monitoring/grafana
-    version: v1.5.0
+    version: v1.6.0-rc1
   - name: monitoring/goldpinger
-    version: v1.5.0
+    version: v1.6.0-rc1
```

Comment:
I think we can avoid updating docs with the rc versions... at least if it's not the result of a super aggressive sed

Reply:
It's automated with bumpversion. So it does not matter if is a rc version or a final release. I can take a look for "bumpversion" conditions ;) let me see

Reply:
Done!
Now it will template the "final" release even if we create a release candidate.